daemon/tests/harness/mod.rs

```diff
@@ -153,6 +162,10 @@ impl Taker {
         &mut self.system.order_feed_receiver
     }
 
+    pub fn maker_status_feed(&mut self) -> &mut watch::Receiver<ConnectionStatus> {
+        &mut self.system.maker_online_status_feed_receiver
+    }
```

Comment:
Is this used?

Reply:
Seems to belong to the patch after this.

Reply:
moved into the correct patch.